### ISSUE TITLE
Processing Uranus Crystal ore - More processing options, please???

### ISSUE THREAD

**Issue by Danekjovax** [Freebi85/Dungeons-Dragons-Space-Shuttles#242]
In JEI, I see the Industrial Blast Furnace (Tech Reborn) being the only way to process Uranus Crystal ore ingots.  Even with all the overclocking upgrades (and requisite transformer upgrades, the best rate I can process ingots is about 20 seconds/ingot.  This is unreasonable for the following reasons:

1) Given the requirements to make a T8 rocket (which you'll need to craft soon after acquiring the ore), you calculate it will take roughly 1500 crystal ingots to process into the required T8 Heavy Duty Plates to complete the rocket.  Calculating the time from here: 1500 * 20 = 30000 seconds / 3600 = 8 hours 20 minutes!  Assuming you only want to make one Ind. Blast Furnace at your base, this is an unreasonably long time.  I'd say the only wait longer than this for something is getting from T1 to T2 Void Ore Miner.  Given that my time to game is very limited (I work a job and am a dad of two kids), I'd like to spend my gaming time doing more than just waiting on one thing to process in my base (yes, I multitask, but still...).

2) Given the plethora of other crystals in the modpack (nether/black quartz, diamond, ruby/sapphire, etc.) that have multiple processing paths for their respective ores, why can't this crystal ore also share the same options?

In conclusion, please allow for more processing paths (pulverizer, industrial grinder, crusher) to allow more options in processing this ore.  

Thanks!

**Comment by Freebi85** [comment]
I will add an alternative recipe for the processing of Uranus Crystal Ore that requires infected water or organic slurry in conjunction with another machine. Still not that easy^^

This modpack is designed to have multiple machines for processing lines as an expert pack. Especially in your current advance for a T8 rocket. You should also set up several Carpenter and Rolling machines for easy automation. 
Of course you can solve everything with only one machine but due to the time costs it is not recommended.
Furthermore it becomes more resource-intensive from here on. I'm surprised that you are so far advanced and only have one BlastFurnace :D

No offense but "(I work a job and am a dad of two kids)" is not a reason for me to change a pack mechanic or a reason at all to request an Enhancement. At least that's how I feel so I don't want to be mean or anything :S

Of course, other resources are more easily accessible through MysticalAg. but that doesn't apply to every ore in the pack. Especially these from space.

**Comment by Danekjovax** [comment]
Thanks for your response, Freebi.

Since my original post, I found several problems with my set up which when resolved (I realized 16k RF/tick was still not enough power, so I upgraded to cryo flux ducts and also upgraded my induction matrix) I was able to increase crystal production to about 2.5sec/ingot, so was able to get around the bottleneck for the most part.

I still believe that multiple ways to process this material would help, regardless.

Again, thanks.

**Comment by Freebi85** [comment]
I think i will keep the new processing system im working on. i think gonna use this fore more stuff in the future to double and triple special items. Thank you for the Feedback.
